apps/codeforafrica/src/lib/data/blockify/index.js

```diff
@@ -3,15 +3,15 @@ import hero from "./hero";
 import meetOurTeam from "./meetOurTeam";
 import ourImpact from "./our-impact";
 import ourTeam from "./ourTeam";
-import stories from "./stories";
+import posts from "./posts";
 
 const propsifyBlockBySlug = {
   "get-involved": getInvolved,
   hero,
   "meet-our-team": meetOurTeam,
   "our-impact": ourImpact,
-  stories,
   "our-team": ourTeam,
+  "post-list": posts,
```

Comment:
I think I asked, why `post-list` instead of just `posts`?

Reply:
@kilemensi Because we are using the slug set in the CMS to match blocks. Payload does not allow us to use similar slugs. We already used `posts` slug for the [Posts collection](https://github.com/CodeForAfrica/ui/blob/25a20dbaffe1f026eff99bf1b519a6ccd4679ab8/apps/codeforafrica/src/payload/collections/Posts.js#L8) so we used [post-list](https://github.com/CodeForAfrica/ui/blob/25a20dbaffe1f026eff99bf1b519a6ccd4679ab8/apps/codeforafrica/src/payload/blocks/Posts.js#L13) for the block

Reply:
... again, have added a comment on the block explaining this?

Reply:
@kilemensi Not sure I follow. This is the only [comment](https://github.com/CodeForAfrica/ui/pull/617#discussion_r1340047858) I can see on the related block

Reply:
I mean: Can you add a single line comment on the block explaining why we're not naming it posts?